src/engine/bank03.asm

```diff
@@ -5755,105 +5021,63 @@ Script_Jonathan: ; eefd (3:6efd)
 
 Script_Ken: ; ef22 (3:6f22)
 	start_script
-	run_command ScriptCommand_TryGivePCPack
-	db $09
-	run_command ScriptCommand_JumpIfFlagNonzero2
-	db EVENT_FLAG_23
-	dw .have_300_cards
-	run_command ScriptCommand_CheckRawAmountOfCardsOwned
-	dw 300
-	dw .have_300_cards
-	run_command ScriptCommand_JumpIfFlagZero1
-	db EVENT_FLAG_24
-	dw NO_JUMP
-	run_command ScriptCommand_PrintVariableText
-	tx Text06ba
-	tx Text06bb
-	run_command ScriptCommand_SetFlagValue
-	db EVENT_FLAG_24
-	db $01
-	run_command ScriptCommand_QuitScriptFully
+	try_give_pc_pack $09
```

Comment:
Do we have constants for this argument? I'm actually not totally sure what it's supposed to represent.

Reply:
I'm not sure if I ever uncovered a table relating to this or not, and I have no idea how entries would be named. It represents which of the 15 packs Dr. Mason will send to your PC.

Reply:
Ohh okay, right. Not a high priority right now then.